app/views/OverviewPageView.scala.html

```diff
@@ -69,24 +69,24 @@ <h2 class="app-task-list__section">
                 </li>
 
             @if(appConfig.employmentReleased){
-            <li class="app-task-list__item">
-                        <span class="app-task-list__task-name">
-                            @if(appConfig.employmentEnabled && incomeSources.employment.isDefined){
-                                <a class="govuk-link" id="employment_link" href="./view">
-                                    @messages("common.employment")
-                                </a>
-                            } else {
+                <li class="app-task-list__item">
```

Comment:
There are multiple execution paths here which render different things so I would expect us to have view tests for this, but i noticed we don't have much even before this ticket. Is this something we want to do? or leave as is and eyeball it?